src/pal/inc/rt/safecrt.h

```diff
@@ -136,7 +136,7 @@ typedef _W64 unsigned int   uintptr_t;
 #define _UINTPTR_T_DEFINED
 #endif
 
-#if defined(_MSC_VER) || (defined(__GNUC__) && (__GNUC__ >= 3))
+#if defined(_MSC_VER) || (defined(__GNUC__) && (__GNUC__ >= 3) && !(defined(PLATFORM_UNIX) && defined(__aarch64__)))
 #define SAFECRT_DEPRECATED __declspec(deprecated)
```

Comment:
It would be better to delete SAFECRT_DEPRECATED and errcode that it is used on instead.